tests/analyse/test_scenario.py

```diff
@@ -846,6 +847,62 @@ def test_model_modelled_obs_ch4(model_scenario_ch4_dummy, footprint_dummy, flux_
     modelled_mf = combined_dataset["mf_mod"].values
     assert np.allclose(modelled_mf, expected_modelled_mf)
 
+@pytest.fixture
+def obs_dummy():
+    """
+    Create example ObsData object with dummy data
+     - Species is methane (ch4)
+     - Hourly frequency for 2012-01-01 - 2012-01-02 (48 time points)
+     - "mf" values are from 1, 48
+    """
+    from openghg.dataobjects import ObsData
+
+    time = pd.date_range("2011-11-04T00:00:00", "2011-11-07T23:00:00", freq="H")
+
+    ntime = len(time)
+    values = np.arange(0, ntime, 1)
+
+    species = "ch4"
+    site = "TEST_SITE"
+    inlet = "10m"
+    sampling_period = "60.0"
+
+    attributes = {
+        "species": species,
+        "site": site,
+        "inlet": inlet,
+        "sampling_period": sampling_period,
+    }
+
+    data = xr.Dataset({"mf": ("time", values)}, coords={"time": time}, attrs=attributes)
+
+    # Potential metadata:
+    # - site, instrument, sampling_period, inlet, port, type, network, species, calibration_scale
+    #   long_name, data_owner, data_owner_email, station_longitude, station_latitude, ...
+    # - data_type
+    metadata = attributes
+    metadata["object_store"] = "/tmp/test-store-123"
+
+    obsdata = ObsData(data=data, metadata=metadata)
+
+    return obsdata
+
+@pytest.fixture
+def model_scenario_dummy(obs_dummy, footprint_dummy, flux_ch4_dummy, bc_ch4_dummy):
```

Comment:
This doesn't need to be a fixture and can just be created in the test.

Reply:
done